### PR TITLE
docs(Scheduler): add hide all day parameter

### PR DESCRIPTION
**Pull request by xristianstefanov**
closes: https://github.com/telerik/blazor/issues/9892

### CODE REVIEW

File: components/scheduler/views/timeline.md [hunk 1]
```diff
@@ -19,9 +19,22 @@ In this article:
...
+## View Parameters
+
+Generally, the views are designed around the timeframe that they show and the day-based views share some common properties that you will likely have to set to provide a good user experience for the user:
+
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
+
+| Attribute | Type and Default Value | Description |
+|----------|----------|----------|
 | `ColumnWidth` | `decimal` | The width of each time column in pixels.
+| `EndTime` | `DateTime` | The counterpart to `StartTime` - defines when the full day ends. Defaults to midnight. If you have the day end earlier you can reduce the amount of elements that render, but the user may not see some late appointments.
```
Comment (yordan-mitev) [inline components/scheduler/views/timeline.md:31]: * Use an em dash.
* "If you have the day end earlier" -> If you set the `EndTime` earlier than midnight, you reduce the elements that render, but the user may not see late appointments.

File: components/scheduler/views/timeline.md [hunk 2]
```diff
@@ -19,9 +19,22 @@ In this article:
...
 | `NumberOfDays` | `int` <br/> `1` | How many days to show side by side in the view.
+| `SlotDivisions` |`int` | The number of partitions in each major time slot.
+| `SlotDuration` | `int` | The time span of each major time slot in minutes.
+| `StartTime` | `DateTime` | This is the first hour that is shown in the view. Defaults to midnight, so if you do not set it to a value close to the start of the working day, the user is likely to see only blank spaces before they scroll down.
```
Comment (yordan-mitev) [inline components/scheduler/views/timeline.md:35]: "This is the first hour..." -> The first hour...

File: components/scheduler/views/timeline.md [hunk 3]
```diff
@@ -19,9 +19,22 @@ In this article:
...
+|`WorkDayEnd` | `DateTime` | The counterpart to `WorkDayStart` - defines when the working day ends.
```
Comment (yordan-mitev) [inline components/scheduler/views/timeline.md:36]: Use an em dash

File: components/scheduler/views/timeline.md [hunk 4]
```diff
@@ -19,9 +19,22 @@ In this article:
...
+| `WorkDayStart` | `DateTime` | This is when the working day starts. The work hours have a different background than non-working hours so the user can distinguish them easily. This parameter also influences the "Show Business Hours" toggle.
```
Comment (yordan-mitev) [inline components/scheduler/views/timeline.md:37]: - "This is when the working day starts." -> The start time of the working day
- Render UI elements in bold: "Show Business Hours" -> **Show Business Hours**

**Review by Tsvetomir-Hr — APPROVED** [review]

**Pull request merged**